modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-rsyslog_{context}"]
+= Configuring the Fluentd remose-syslog plug-in to send logs to an external log aggregator
+
+You can configure Fluentd to send a copy of its logs to an external Rsyslog server, 
+and not the default Elasticsearch, using the *fluent-plugin-remote-syslog* plug-in.
+
+[NOTE]
+====
+This legacy *fluent-plugin-remote-syslog* method is deprecated and will be removed in a future release.
+====
+
+There are two versions of the *fluent-plugin-remote-syslog* plug-in:
+
+* *out_syslog*. Use `syslog` in the ConfigMap for the non-buffered implementation, communicating through UDP. 	
+This implementation is insecure, and should only be used in environments where you can guarantee no snooping on the connection.
+* *out_syslog_buffered*: Use `syslog_buffered` in the ConfigMap for the buffered implementation, communicating through TCP.
+
+To configure the *fluent-plugin-remote-syslog* plug-in, create a ConfigMap called `secure-forward` in the `openshift-logging` namespace. 
```

Comment:
secure-forward?

Reply:
@richm Thanks for the pointers here. I repurposed the secure-forward topic for consistency between the two. Still have some clean-up to do. Just wanted to make sure I was going in the right direction with this.

Reply:
@richm @jcantrill Does the user create a configmap from a configuration file to use this plugin? If so, do they need specific names?